graze/pistachio
# adds new markup for feature blocks and documentation

**Summary**

Refactors existing feature banners

Adds several new feature layout options:
- feature strips
- feature full width banner
- feature images

**Notable Changes**

existing feature banners have been refactored to avoid css spaghetti - this is a breaking change and will require new markup

**QA**

Check all new feature options in multiple browsers and devices. Make sure text is legible and everything is roughly consistent visually.

**Screenshots**

![pistachio---layouts-feature-block-examples](https://cloud.githubusercontent.com/assets/713293/20630218/9ab05d1c-b327-11e6-8234-a92beee00274.jpg)
